src/test/scala/org/broadinstitute/dsde/workbench/sam/api/ResourceRoutesSpec.scala

```diff
@@ -29,7 +29,7 @@ class ResourceRoutesSpec extends FlatSpec with Matchers with ScalatestRouteTest
     new TestSamRoutes(resourceTypes, mockResourceService, mockUserService, userInfo)
   }
 
-  "ResourceRoutes" should "404 for unknown resource type" in {
+  "GET /api/resources/{resourceType}/{resourceId}/actions/{action}" should "404 for unknown resource type" in {
```

Comment:
this is reorganized to help visualize which tests belong to which endpoints

sample output from a test run:

```
[info] ResourceRoutesSpec:
[info] GET /api/resources/{resourceType}/{resourceId}/actions/{action}
[info] - should 404 for unknown resource type
[info] GET /api/resourceTypes
[info] - should 200 when listing all resource types
[info] POST /api/resources/{resourceType}/{resourceId}
[info] - should 204 create resource
[info] GET /api/resources/{resourceType}/{resourceId}/roles
[info] - should 200 on list resource roles
[info] - should 404 on list resource roles when resource type doesnt exist
[info] PUT /api/resources/{resourceType}/{resourceId}/policies
[info] - should 201 on a new policy being created for a resource
[info] - should 201 on a policy being updated
[info] - should 400 on a policy being created with invalid actions
[info] - should 400 on a policy being created with invalid roles
[info] - should 403 when creating a policy on a resource when the user doesn't have alterpolicies permission (but can see the resource)
[info] - should 404 when creating a policy on a resource type that doesnt exist
[info] - should 404 when creating a policy on a resource that the user doesnt have permission to see
[info] GET /api/resources/{resourceType}/{resourceId}/policies
[info] - should 200 when listing policies for a resource and user has readpolicies permission
[info] - should 403 when listing policies for a resource and user lacks readpolicies permission (but can see the resource)
[info] - should 404 when listing policies for a resource type that doesnt exist
[info] - should 404 when listing policies for a resource when user can't see the resource
[info] DELETE /api/resources/{resourceType}/{resourceId}
[info] - should 204 when deleting a resource and the user has permission to do so
[info] - should 403 when deleting a resource and the user has permission to see the resource but not delete
[info] - should 404 when deleting a resource of a type that doesn't exist
[info] - should 404 when deleting a resource that exists but can't be seen by the user
```